### DISCUSSION TITLE
use Filament with existing application

### DISCUSSION THREAD

**Discussion by AutobookNft** [filamentphp/filament#9641] [Help]
My Laravel v10 application, livewire 3.0, Filament 3.0, still runs locally, but I have a database with test data. The users table also has a slightly different field naming than the one created by Filament, for example first_name and not name. So I would need to understand how to interact on a deeper level with Filament scaffolding

**Comment by leandrocfe** [comment]
Most likely, you'll need to make some adjustments, [for example](https://filamentphp.com/docs/3.x/panels/users#configuring-the-users-name-attribute), to the code used in your current application, but rest assured, it's entirely feasible. Alternatively, you could opt for a fresh installation, migrate your tables/models, and then create or generate the necessary resources.

**Reply by AutobookNft** [reply]
I found this answer in the documentation, exactly what I needed

![Screenshot 2023-11-15 155514](https://github.com/filamentphp/filament/assets/100548780/fce84323-12c8-4e42-a4dd-93bb1d0383d3)

**Reply by leandrocfe** [reply]
> I found this answer in the documentation, exactly what I needed
> 
> ![Screenshot 2023-11-15 155514](https://github.com/filamentphp/filament/assets/100548780/fce84323-12c8-4e42-a4dd-93bb1d0383d3)
> 

The same link on my answer 👍
